com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/common/JsonResponse.java

```diff
@@ -32,9 +32,26 @@
 
 public class JsonResponse {
 	
-	private static ObjectNode type(String type) {
-		// Allowed types: success, error, fullUpdate, incrementalUpdate
-		return Json.object(Json.prop("type", Json.text(type)));
+	private static ObjectNode type(JsonResponseType type) {
+		String typeString = "";
```

Comment:
Instead of mapping this manually, create a constructor for the enum and override `toString` method on it, such that we can call `type.toString` here.